### PR TITLE
updates to docs homepage, unifying pages homepages components under p…

### PR DESCRIPTION
**Pull request by jakepage91**
Fixed the light mode image on the docs overview page. 
Revamped docs landing page

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ sidebar_label: Overview
 ---
 # Overview of Komiser Documentation
 
-![Tailwarden](/img/docs-banner-tw.png)
+
+![Tailwarden](/img/komiser-overview-light.png#gh-light-mode-only)![Tailwarden](/img/komiser-overview-dark.png#gh-dark-mode-only)
```
Comment (mlabouardy) [inline docs/intro.md:9]: is it possible to use if else conditions here to show an image only if dark or light mode is enabled. If not, I would just keep the light version only and remove the dark mode version

File: docs/intro.md [hunk 2]
```diff
@@ -5,7 +5,9 @@ sidebar_label: Overview
...
+
 [![Twitter URL](https://img.shields.io/twitter/url/https/twitter.com/fold_left.svg?style=social&label=Follow%20%40mlabouardy)](https://twitter.com/mlabouardy) [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Optimize%20Cost%20and%20Security%20on%20AWS&url=https://github.com/mlabouardy/komiser&via=mlabouardy&hashtags=komiser,aws,gcp,cloud,serverless,devops)
```
Comment (mlabouardy) [inline docs/intro.md:11]: can you change from my twitter account to tailwarden account

**Review by mlabouardy — CHANGES_REQUESTED** [review]

File: src/pages/index.js [hunk 3]
```diff
@@ -3,10 +3,73 @@ import clsx from 'clsx';
...
+        Komiser enables you to have a clear view into your cloud account, gives helpful advice to reduce the cost and secure your environment.
+      </>
+    ),
+    imageUrl: 'img/focus-on-what-matters.png'
+  },
+  {
+    title: 'The Future of DevOps',
+    description: (
+      <>
+        Komiser is the first step in our plan to create the future generation of Developers and DevOps engineers. The future of DevOps can only be built with the understanding of what we actually have in our cloud accounts, and Komiser is here to help with exactly that.
```
Comment (mlabouardy) [inline src/pages/index.js:35]: Just rephrased to keep it short and aligned "The future of DevOps start with understanding of what we actually have in our cloud accounts, and Komiser can you help discover your resources.", feel free to tweak the wording ofc

Screenshot 2023-01-19 at 12 44 19

**Review by mlabouardy — CHANGES_REQUESTED** [review]
Thanks for the changes, dropped few improvements

File: docs/intro.md (hunk 1 above)
Comment (jakepage91) [reply docs/intro.md:9]: Opps, I added back the css that automatically detects the mode we are in, so without requiring an if else statement, we will only we showing one image at a time now.

File: docs/intro.md (hunk 2 above)
Comment (jakepage91) [reply docs/intro.md]: Done

File: src/pages/index.js (hunk 3 above)
Comment (jakepage91) [reply src/pages/index.js]: I tweaked it a little to "The future of DevOps starts with knowing exactly what we have in our cloud accounts, and Komiser can you help discover every last one of your resources."
Cool?

**Pull request merged**